From e762e181526596cd1e8a7091ae2e69f18a7860cc Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 19 Dec 2023 13:54:33 +0100
Subject: [PATCH 0763/1141] dt-bindings: crypto: stm32: add 'power-domains'
 property

STM32 CRYP may be in a power domain which is the case for the STM32MP25
 based boards. Allow a single 'power-domains' entry for STM32 CRYP.

Fix dtbs_check warning:
Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I7843cce28b8fd276e414a32b40f937057d48783e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347864
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 Documentation/devicetree/bindings/crypto/st,stm32-cryp.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/crypto/st,stm32-cryp.yaml b/Documentation/devicetree/bindings/crypto/st,stm32-cryp.yaml
index ed23bf94a8e0..59ec8d3b1afc 100644
--- a/Documentation/devicetree/bindings/crypto/st,stm32-cryp.yaml
+++ b/Documentation/devicetree/bindings/crypto/st,stm32-cryp.yaml
@@ -27,6 +27,9 @@ properties:
   resets:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
 required:
   - compatible
   - reg
-- 
2.39.2

